tests/phpunit/tests/rest-api/rest-block-type-controller.php

```diff
@@ -551,6 +559,8 @@ public function test_get_item_schema() {
 		$this->assertArrayHasKey( 'textdomain', $properties );
 		$this->assertArrayHasKey( 'name', $properties );
 		$this->assertArrayHasKey( 'attributes', $properties );
+		$this->assertArrayHasKey( 'editor_selectors', $properties );
+		$this->assertArrayHasKey( 'selectors', $properties );
```

Comment:
* Add @ticket 57585 to the test method to identify this change for tacking.
* Add an assertion failure message to each assertion please.